src/azul/service/manifest_service.py

```diff
@@ -604,9 +608,19 @@ def entity_type(self) -> str:
     def source_filter(self) -> SourceFilters:
         return [
             *super().source_filter,
-            'contents.files.related_files'
+            'contents.files.related_files',
+            'contents.files.drs_path'
         ]
 
+    @cached_property
+    def manifest_config(self) -> ManifestConfig:
```

Comment:
Why don't you just statically modify the config literal in the plug-in? The injection into the hit could then be made conditonal based on the absence/presence of that config entry.